.vscode/extensions.json

```diff
@@ -0,0 +1,5 @@
+{
+    "recommendations": [
+        "mccarter.start-git-bash"
+    ]
+}
```

Comment:
This change is not related to #6 or #9. I suggest don't make changes to `.vscode/extensions.json`